hwrt/download.py

```diff
@@ -14,11 +14,22 @@
 # hwrt modules
 import hwrt.utils as utils
 
+def generate_file_md5(local_path_file, blocksize=2**20):
+    # http://stackoverflow.com/a/1131255/3345926
+    m = hashlib.md5()
+    with open( local_path_file , "rb" ) as f:
+        while True:
+            buf = f.read(blocksize)
+            if not buf:
+                break
+            m.update( buf )
```

Comment:
Could you please remove the whitespace (-> `m.update(buf)`) as described in PEP8?